### ISSUE TITLE
how to jump from one action to another?

### ISSUE THREAD

**Issue by maxchen-000** [hubtype/botonic#2649]
I have an action that make an api call that take several seconds or even minute. How do I display a text telling user something like "calling backend api, this will take few seconds please wait..." and then move on to make the api call?
Based on the documentation, the "render" function in the action is for sending the output to the user. How do I send the render and then move on to the next action?
basically, something like send message to user, then call botonicInit, then call render.

**Comment by maxchen-000** [comment]
Found a possible workaround for sending a message before render is to call the Botonic.addBotText() in the botonicInit() function before the api call something like:
`
Botonic.addBotText("please wait")     

// make the api call and return the result 
`

Still need to figure out how to call another action.

**Comment by maxchen-000** [comment]
it looks like we just need to use Botonic.addUserPayload() in the botonicInit() to jump to another action.
Something like:
`
Botonic.addUserPayload("next-action")
`
